clients/fides-js/src/components/fides.css

```diff
@@ -5,6 +5,7 @@
   /* Colors */
   --fides-overlay-primary-color: #8243f2;
   --fides-overlay-background-color: #f7fafc;
+  --fides-embedded-overlay-background-color: white;
```

Comment:
```suggestion
  --fides-overlay-embed-background-color: transparent;
```

Reply:
How about we do this instead? Transparent means that by default this'll show the color of the page

Reply:
oh this makes sense. Lemme test and 🚢  it!